src/coreclr/vm/exceptmacros.h

```diff
@@ -358,11 +357,9 @@ VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex, bool isHar
         bool       __fExceptionCaught = false;                                             \
         SCAN_EHMARKER();                                                                    \
         if (true) PAL_CPP_TRY {                                                             \
-            SCAN_EHMARKER_TRY();                                                            \
-            DEBUG_ASSURE_NO_RETURN_BEGIN(IUACH);
```

Comment:
Nothing in the enclosing block in the `INSTALL` macros nor any code following in the `UNINSTALL` macros depend on us not returning from within this block.